src/Treasure/Tui/Chest.hs

```diff
@@ -0,0 +1,61 @@
+{-# LANGUAGE Rank2Types #-}
+
+{- | This module contains data types to work with application form.
+'TreasureChest' is the data type containing the values manipulated by the fields
+in the form.
+-}
+
+module Treasure.Tui.Chest
+       ( -- * Data types
+         TreasureChest (..)
+       , CheckBox (..)
+       , toCheckBoxes
+
+         -- * Lenses
+       , chestAccountsL
+       , checkBoxL
+       ) where
+
+import Control.Lens (Lens', ix, lens, (.~))
```

Comment:
I noticed that `brick` depends on the `microlens` library. I was thinking earlier that `lens` can help us better but now I see that everything we need is in `microlens` and `microlens-th` packages. So we probably can depend on `microlens-*` libraries to make `treasure-keeper` slightly smaller and build faster